src/components/CodeInput/inputItems.js

```diff
@@ -20,11 +21,19 @@ const InputItems = React.forwardRef((props, ref) => {
         id,
     } = props;
 
+    const handleOnFocus = e => {
+        if (readOnly || disabled) {
+            ref.current.blur();
+        } else {
+            onFocus(e);
+        }
+    };
+
     const inputs = value.map((inputValue, index) => {
         const inputIndex = index;
         const isActive = inputIndex === focusedIndex;
         const inputRef = isActive ? ref : null;
-        const inputId = isActive ? id : null;
+        const inputId = `${id}-${index}`;
```

Comment:
as said before input id should be generated with uniqueId, but since we will not associate it with no label then it is not needed then, for now, let's not pass id to the input